pkg/adapter/v2/cloudevents.go

```diff
@@ -67,6 +79,30 @@ func NewCloudEventsClientCRStatus(target string, ceOverrides *duckv1.CloudEventO
 	}, nil
 }
 
+func withTimeOut(duration time.Duration) http.Option {
```

Comment:
@n3wscott @slinkydeveloper does it make sense to move this to the CE SDK?

Reply:
Not sure we need it, we already have `WithClient` (which you can use it here, more than manually creating withTimeout method)